test/unit/useClientRequest.test.js

```diff
@@ -73,25 +73,112 @@ describe('useClientRequest', () => {
   });
 
   describe('fetchData', () => {
-    it.skip('needs testing', () => {
-      // fetchData updates the state & therefore needs to be wrapped inside an `act`
```

Comment:
I would probably leave this comment in. Someone is inevitably going to look at this test when they see it kicking out warnings so it'd be good to leave this info here.

Reply:
Good shout